### PR TITLE
Replace deprecated Form constants with the new ones

### PR DESCRIPTION
**Pull request by franmomu**
## Subject

Replacing deprecated operator constants.

I am targeting this branch, because this is BC.

## Changelog

```markdown
### Fixed
- Replace deprecated operator constants.
```

### CODE REVIEW

File: src/Filter/StringFilter.php [hunk 1]
```diff
@@ -67,7 +67,7 @@ public function getDefaultOptions()
 
     public function getRenderSettings()
     {
-        return [ChoiceType::class, [
+        return [ContainsOperatorType::class, [
```
Comment (jordisala1991) [inline src/Filter/StringFilter.php:70]: This one has to be changed?

File: src/Filter/ModelFilter.php [hunk 2]
```diff
@@ -55,7 +55,7 @@ public function getDefaultOptions()
             'field_name' => false,
             'field_type' => DocumentType::class,
             'field_options' => [],
-            'operator_type' => EqualType::class,
+            'operator_type' => EqualOperatorType::class,
```
Comment (jordisala1991) [inline src/Filter/ModelFilter.php:58]: This type change is intentional?

File: src/Guesser/FilterTypeGuesser.php [hunk 3]
```diff
@@ -60,7 +60,7 @@ public function guessType($class, $property, ModelManagerInterface $modelManager
             switch ($mapping['type']) {
                 case ClassMetadata::ONE:
                 case ClassMetadata::MANY:
-                    $options['operator_type'] = EqualType::class;
+                    $options['operator_type'] = EqualOperatorType::class;
```
Comment (jordisala1991) [inline src/Guesser/FilterTypeGuesser.php:63]: this change is correct?

File: src/Filter/StringFilter.php (hunk 1 above)
Comment (franmomu) [reply src/Filter/StringFilter.php]: Nope, thanks!

File: src/Filter/ModelFilter.php (hunk 2 above)
Comment (franmomu) [reply src/Filter/ModelFilter.php:58]: In this case and the one below it is, [EqualType is deprecated](https://github.com/sonata-project/form-extensions/blob/1.x/src/Type/EqualType.php#L20).

Comment (VincentLanglet) [reply src/Filter/ModelFilter.php:58]: You can compare with https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/3.x/src/Filter/ModelFilter.php#L49 @jordisala1991

File: src/Guesser/FilterTypeGuesser.php (hunk 3 above)
Comment (VincentLanglet) [reply src/Guesser/FilterTypeGuesser.php:63]: Yes, see https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/3.x/src/Guesser/FilterTypeGuesser.php#L62

**Comment by jordisala1991** [comment]
Thank you @franmomu

**Pull request merged**